packages/preferences/src/browser/folder-preference-provider.ts

```diff
@@ -14,6 +14,8 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
We should not disable `no-explicit-any` for the whole file, but rather just for the line where it is used.